tailwind/tailwind.css

```
@@ -852,3 +852,189 @@ select:disabled,
     }
   }
 }
+
+/*--- Opportunity header -------------------------
```

Stylelint (17.14.1)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)